README.Rmd

```diff
@@ -98,7 +98,19 @@ See 'Writing R Extensions' about including UTF-8 characters in packages: <https:
 
 I suggest that you keep your source files in ASCII, except for comments, which you can write in UTF-8 if you like. (But not necessarily in roxygen2 comments, which will go in the manual, see below!)
 
-If you need to include non-ASCII characters in strings, then use UTF-8 and `\uxxxx` or `\U{xxxxxx}` escape sequences.
+If you need a string containing non-ASCII characters, construct it with `\uxxxx` or `\U{xxxxxx}` escape sequences, which yields a UTF-8-encoded string.
+
+```{r}
+one_and_a_half <- "\u0031\u00BD"
+one_and_a_half
+Encoding(one_and_a_half)
+charToRaw(one_and_a_half)
+```
+
+Here are some handy ways to find the Unicode code points for an existing string:
+
+* Copy and paste into the [Unicode character inspector](https://apps.timwhitlock.info/unicode/inspect).
+* *Do we have other suggestions?*
```

Comment:
How do you do this @gaborcsardi?

Reply:
`cat(stringi::stri_escape_unicode("µ"))` ?

Reply:
Or `utf8::utf8_print("µ", utf8 = FALSE)`

Reply:
Or even base R:

```
utf8ToInt("µ")
#> [1] 181
as.hexmode(utf8ToInt("µ"))
#> [1] "b5"
```

Reply:
I'll add these....